src/api/common/schemas/suspended-payment.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "suspended-payment",
+  "type": "object",
+  "properties": {
+    "source": {"$ref": "maxAdjustment"},
+    "destination": {"$ref": "adjustment"},
+    "memos": {
+      "type": "array",
+      "items": {
+        "$ref": "memo"
+      }
+    },
+    "digest": {"$ref": "hash256"},
+    "cancelAfter": {
+      "type": "integer",
+      "minimum": 0,
+      "description": "milliseconds since unix epoch"
+    },
+    "finishAfter": {
```

Comment:
That's exactly how I originally interpreted the rippled parameters. It wasn't until I started writing tests that I realized that it means transaction can only be in one of two modes (Cancel/Finish) at a time. `allowCancelAfter`/`allowFinishAfter` might be even more clear